README.md

```diff
@@ -28,9 +28,11 @@ https://groups.google.com/a/arduino.cc/forum/#!forum/developers
 
 Installation
 ------------
-Detailed instructions are in reference/Guide_Windows.html and
-reference/Guide_MacOSX.html.  For Linux, see the Arduino playground:
-http://www.arduino.cc/playground/Learning/Linux
+Detailed instructions for installation in popular operating systems.  
+For Linux, see the Arduino playground: http://www.arduino.cc/playground/Learning/Linux      
+For macOS X: https://www.arduino.cc/en/Guide/MacOSX        
+For Windows :https://www.arduino.cc/en/Guide/Windows
```

Comment:
There is a misplaced space there. Also, why not link to https://www.arduino.cc/en/Guide/Linux? If the playground version is better, we should probably improve the Guide version and/or let that link to the playground for details.